core/frontend/src/tile/Tile.ts

```diff
@@ -83,6 +83,7 @@ export abstract class Tile {
   protected _contentRange?: ElementAlignedBox3d;
   /** The maximum size in pixels this tile can be drawn. If the size of the tile on screen exceeds this maximum, a higher-resolution tile should be drawn in its place. */
   protected _maximumSize: number;
+  protected _outOfLodRange?: boolean;
```

Comment:
You never assign to _outOfLodRange outside of the constructor, shouldn't it be read-only?

Reply:
Actually just remove it from `Tile`.